### PR TITLE
Enhanced log messages

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- Enhanced log messages in order to track chaindata fetcher.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: datasync/chaindatafetcher/chaindata_fetcher.go [hunk 1]
```diff
@@ -115,33 +115,37 @@ func (f *ChainDataFetcher) Start(server p2p.Server) error {
...
-	logger.Info("wait for all goroutines to be terminated...")
 	f.wg.Wait()
-	logger.Info("terminated all goroutines for chaindatafetcher")
+	logger.Info("chaindata fetcher is stopped")
 	return nil
 }
 
-func (f *ChainDataFetcher) makeRequests(start, end uint64, reqType requestType, stopCh chan struct{}) {
-	for i := start; i <= end; i++ {
+func (f *ChainDataFetcher) makeRequests(startBlock, endBlock uint64, reqType requestType, stopCh chan struct{}) {
```
Comment (ehnuje) [inline datasync/chaindatafetcher/chaindata_fetcher.go:138]:
```suggestion
func (f *ChainDataFetcher) sendRequests(startBlock, endBlock uint64, reqType requestType, stopCh chan struct{}) {
```

File: datasync/chaindatafetcher/kas/repository_transactions.go [hunk 2]
```diff
@@ -128,7 +129,12 @@ func transformToTxs(event blockchain.ChainEvent) []*Tx {
 
 // InsertTransactions inserts transactions in the given chain event into KAS database.
 func (r *repository) InsertTransactions(event blockchain.ChainEvent) error {
-	return r.insertTransactions(transformToTxs(event))
+	txs := transformToTxs(event)
+	if err := r.insertTransactions(txs); err != nil {
+		logger.Error("Failed to insertTransactions", "err", err, "blockNumber", event.Block.NumberU64(), "numTxs", len(txs))
```
Comment (ehnuje) [inline datasync/chaindatafetcher/kas/repository_transactions.go:134]: `insertTransactions` already leaves logs internally.

File: datasync/chaindatafetcher/chaindata_fetcher.go (hunk 1 above)
Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/chaindata_fetcher.go]: I rebased the PR, and it is resolved. :)

File: datasync/chaindatafetcher/kas/repository_transactions.go (hunk 2 above)
Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kas/repository_transactions.go:134]: The internal logs were removed and the logs are pushed up.

**Pull request merged**